### PR TITLE
Add Contact form in mobile version

### PR DESCRIPTION
**Pull request by sajid-munawar**
- Add contact form
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- The message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,5 +193,65 @@ <h2>About me</h2>
         </li>
       </ul>
     </section>
+
+    <footer>
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
```
Comment (thecodechaser) [inline index.html:201]: - [ ] Please increase the font-weight for this paragraph and make sure it matches the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -193,5 +193,65 @@ <h2>About me</h2>
...
+        <textarea
+          cols="30"
+          rows="10"
+          placeholder="Enter text here"
+          maxlength="500"
+          required
+          name="Message"
+        ></textarea>
+        <button type="submit">Get in touch</button>
+      </form>
```
Comment (thecodechaser) [inline index.html:214]: - [ ] Please use the same width for all form fields and make sure it matches with the Figma design. Right now the message field width is smaller than the other two fields.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @sajid-munawar, and @mr-abdellah 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @sajid-munawar, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**